src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -571,8 +573,9 @@ internal sealed class RSession : IRSession, IRCallbacks {
         /// <param name="url"></param>
         /// <returns></returns>
         Task IRCallbacks.WebBrowser(string url) {
```

Comment:
Calling this methods for all URLs is intentional, because it's also used for Shiny etc. Think of it as a proxy - because the URL is issued by the machine on which the broker and the hosts are running, to interpret it correctly, it may need to be fetched on that machine - not just for `localhost` URLs like Help and Shiny, but also potentially for URLs of other intranet stuff that's only accessible from broker but not from client. Always going through the broker makes sure that it always just works.